docs/pipelines.md

```diff
@@ -791,6 +791,40 @@ Overall, `PipelineRun` state transitioning is explained below for respective sce
 Please refer to the [table](pipelineruns.md#monitoring-execution-status) under Monitoring Execution Status to learn about
 what kind of events are triggered based on the `Pipelinerun` status.
 
+
+### Using Execution `Status` of `pipelineTask`
+
+Finally Task can utilize execution status of any of the `pipelineTasks` under `tasks` section using param:
+
+```yaml
```

Comment:
I forgot if we support `when` expressions on finally pipeline tasks yet? 
That would be a nice example to use here, but probably it's not supported yet?

Reply:
yup that would be great, we dont have when expression support in finally yet. We have a feature request open here https://github.com/tektoncd/pipeline/issues/3438 😄